marlowe-dashboard-client/src/MainFrame/State.purs

```diff
@@ -1,24 +1,30 @@
 module MainFrame.State (mkMainFrame) where
 
 import Prelude
-import Data.Lens (modifying, use)
+import Data.Lens (assign, modifying, use)
 import Data.Maybe (Maybe(..))
 import Effect.Aff.Class (class MonadAff)
 import Halogen (Component, HalogenM, raise)
 import Halogen as H
 import Halogen.HTML (HTML)
-import MainFrame.Types (Action(..), Msg(..), Query(..), State, ChildSlots, _on)
+import MainFrame.Lenses (_card, _on, _overlay, _screen)
+import MainFrame.Types (Action(..), ChildSlots, Msg(..), Query(..), Screen(..), State)
 import MainFrame.View (render)
 import WebSocket (StreamToServer(..))
 
 initialState :: State
-initialState = { on: true }
+initialState =
+  { overlay: Nothing
+  , screen: Home
+  , card: Nothing
+  , notifications: []
+  , templates: []
+  , contracts: []
+  , on: true
```

Comment:
What is the `on` property for?

Reply:
Nothing. It's just a dummy property to check the websocket is working.

Reply:
lets add a small comment in the Types with a `FIXME: remove`